protocol/avdl/stellar1/local.avdl

```diff
@@ -91,11 +91,13 @@ protocol local {
   // OutsideCurrencyCode examples: "USD", "EUR". Has to be one of
   // supported currencies, returned from getDisplayCurrenciesLocal.
   void changeDisplayCurrencyLocal(int sessionID, AccountID accountID, OutsideCurrencyCode currency);
+  OutsideCurrencyCode getDisplayCurrencyLocal(int sessionID, AccountID accountID);
```

Comment:
I tried to imagine what frontend would use this RPC for. Maybe they'd want it to return `FormatCurrencyLabel` as well since they won't format it. Or maybe currency information would just come along for the ride in a bigger RPC, or at the same time as the other choices? But I'm just imagining. Ask frontend if this is useful its current form?

Reply:
Good point. This wasn't in the doc at all because it wasn't on the same page regarding where displayCurrency actually belong (user or account). So I just wanted to add anything. I'll make it return more stuff instead of just code and ask them if it's fine.

Reply:
Don't make it return more stuff just because of my theories about frontend.